draft-ietf-rats-eat.md

```diff
@@ -1208,21 +1214,22 @@ security state of the entity storing the private key used in a PoP application.
 
 # Detached EAT Bundles {#DEB}
 
-A detached EAT bundle is a structure to convey a fully-formed and signed token plus detached claims set that relate to that token.
-It is a top-level EAT message like a CWT or JWT.
-It can be occur any place that CWT or JWT messages occur.
-It may also be sent as a submodule.
+A detached EAT bundle is a structure to convey an EAT plus detached claims sets secured by that EAT.
+It is a top-level message like a CWT or JWT.
+It can be occur any place that a CWT or JWT occurs.
+It may be sent as a submodule.
 
 A detached EAT bundle consists of two parts.
 
-The first part is a full top-level token.
-This top-level token MUST have at least one submodule that is a detached digest.
-This top-level token may be either CBOR or JSON-encoded.
+The first part is an encoded EAT.
```

Comment:
```suggestion
The first part is encoded as a traditional EAT.
```

Reply:
I think "is an encoded EAT" is quite accurate, but a little hard to get your head around.

It can be either JSON or CBOR-encoded. When CBOR is can be either a CWT or a UCCS and it is bstr wrapped. When JSON it is either a JWT or a UJCS. Note that UCCS and UJCS are not defined in this document. It could also be something else defined outside this document.

I tried using the term "full top-level token" to emphasize this, but I think "encoded EAT" is more accurate.

I don't like "traditional EAT" because the first part can be something not defined in the EAT document (e.g., UCCS or other future things (that I hope we don't defined)).

I don't want to change the wording here because I think it is precise and clean and the CDDL unfolds all the options here.